src/themes/teams/components/Avatar/avatarStyles.ts

```diff
@@ -56,9 +56,7 @@ export default {
     height: pxToRem(getAvatarDimension(size)),
     width: pxToRem(getAvatarDimension(size)),
   }),
-  imageAvatar: ({ props: { size } }) => ({
-    width: pxToRem(getAvatarDimension(size)),
-    height: pxToRem(getAvatarDimension(size)),
+  imageAvatar: () => ({
     verticalAlign: 'top !important',
```

Comment:
😕 hmmm, let's not use important if we can avoid it.  Betting this was copied cruft from SUIR?  Let's try removing it please.